e2e/app/page/workspaces-page.ts

```diff
@@ -110,7 +110,7 @@ export default class WorkspacesPage extends AuthenticatedPage {
     const editPage = await this.clickCreateNewWorkspace();
     // wait for Billing Account default selected value to appear
     const selectBilling = editPage.getBillingAccountSelect();
-    await selectBilling.waitForSelectedValue(UseFreeCredits);
+    await selectBilling.waitForSelectedValue(UseFreeCredits, 60000);
```

Comment:
Sometimes, the `Billing Account` Select dropdown is very slow to get value. It takes longer than 30 sec to show the default value `'Use All of Us free credits'`.